voltdbclient/url_test.go

```diff
@@ -0,0 +1,50 @@
+package voltdbclient
+
+import (
+	"testing"
+)
+
+func TestParseURL(t *testing.T) {
+	sample := []struct {
+		conn                       string
+		pass                       bool
+		scheme, user, passwd, host string
+	}{
+		{"https://localhost:21212", true, "https", "", "", "localhost:21212"},
+		{"http://localhost:21212", true, "http", "", "", "localhost:21212"},
+		{"localhost:21212", true, "localhost", "", "", "localhost:21212"},
+		{"127.0.0.1:21212", false, "voltdb", "", "", "localhost:21212"},
```

Comment:
The playground example you point to is actually valid. I tried the following forms and they are all valid.
- 127.0.0.1
- voltdb://127.0.0.1
- voltdb://127.0.0.1:21212

However, the following form is invalid because it was trying to parse the IP as scheme.
- 127.0.0.1:21212

I think it's important to test and support valid forms with IP addresses because that is quite common.